fixtures/js-stubs/span.ts

```diff
@@ -1,11 +1,14 @@
-export function Span(params = {}) {
+import {RawSpanType} from 'sentry/components/events/interfaces/spans/types';
```

Comment:
@gggritso mind just doing import {type} here so it gets stripped when we run actual tests? It should keep the mock imports isolated and end the require chains in each of these fixture files

Reply:
Ah yes, sorry, I forgot again!

Reply:
No prob, already fixed!